web-app/src/pages/Dashboard/container.js

```diff
@@ -198,6 +180,54 @@ const DashboardContainer = Wrapped =>
       });
     };
 
+    handleFormStatus(name, value, formIsValid) {
+      const formData = { ...this.state.booking.formData };
+      formData[name] = value;
+
+      if (name == 'startDate' || name == 'endDate') {
+        formData.isHalfday = false;
+        formData.isWFH = false;
+      }
+
+      if (name == 'startDate' && formData.startDate.isAfter(formData.endDate)) {
```

Comment:
same again, if practical triple equals across the file.

Reply:
updated to triple equals.